app/templates/macros/nav_macros.html

```diff
@@ -84,10 +123,8 @@ <h3 class="ui header">Bulk</h3>
 
         {# The menu items which will be shown when open-nav is clicked #}
         <div class="ui fixed vertical fluid menu">
-            {% if endpoints %}
```

Comment:
OH wait i figured it out: the fixed nav is only in the m4a-jlc-sp2 branch because I only fixed the dropdown for the user side (we don't have a dropdown for static pages in the generalized product) --this version of the file is messed up, but i reset it to the master version and i think it should be good now.